nft-api/minting.md

```diff
@@ -7,35 +7,38 @@ authors:
     email: artur@licenserocks.de
     avatar: ../static/artur.jpeg
     link: https://twitter.com/ArtiChmaro
+  - name: Adam Leszczyński
+    email: adam.leszczynsky@gmail.com
+    link: https://alsd.app
```

Comment:
change it to your twitter/linkedin if you like, our docs are not advertising platform